### PR TITLE
User can login to account

### PR DESCRIPTION
**Pull request by RhonGabriel**
# PT
link https://www.pivotaltracker.com/story/show/160581978

# User story
```
As a user
In order to access my account
I would like to be able to log in to my existing account
```
```
Create feature file name “login_to_account.feature”
Create a file in step definitions called "login_to_account_steps.rb"
Create FactoryBot
```
# No gems

### CODE REVIEW

File: features/step_definitions/login_to_account_steps.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Given("I am on the landing page") do
+    visit root_path
+end
+  
+When("I click on {string}") do |string|
+    click_on string
+end
+  
+When("I fill in {string} with {string}") do |string, string2|
```
Comment (tochman) [inline features/step_definitions/login_to_account_steps.rb]: Please name the variables

File: features/login_to_account.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Feature: User can login to account
+    
+    As a User,
+    I visit the application,
```
Comment (tochman) [inline features/login_to_account.feature]: Check the format of this user story. I sense a missing ‘when’

Comment (tochman) [reply features/login_to_account.feature]: But also, why would the user want to log in. What’s the purpose (business objective)?

**Review by tochman — DISMISSED** [review]

File: features/login_to_account.feature [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    I would like to be able to log in to my existing account
```
Comment (SophieSjoberg) [inline features/login_to_account.feature]: You have to revisit this user story - how do you write them? Thinking about line 4 here.

File: features/login_to_account.feature [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+Background: 
+    Given User has an existing account
```
Comment (SophieSjoberg) [inline features/login_to_account.feature]: Would probably rephrase this to make it more specific. Given the following user exist

File: features/step_definitions/login_to_account_steps.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    fill_in(string, with: string2)
+end
+  
+Then("I should see message {string}") do |message|
+    expect(page).to have_content message
+end
```
Comment (SophieSjoberg) [inline features/step_definitions/login_to_account_steps.rb:21]: We do not name the variables string or string2. Change those and be specific

**Review by SophieSjoberg — CHANGES_REQUESTED** [review]
Well done, see and adress my comments please 👍

**Comment by RhonGabriel** [comment]
@tochman @SophieSjoberg thanks for the suggestions. I modified the user story to state its purpose and gave name to generic variables

**Review by SophieSjoberg — APPROVED** [review]
